docker-compose.yml

```diff
@@ -1,8 +1,10 @@
-version: "2"
+version: "2.1"
```

Comment:
Why the change to 2.1? I did a quick review of 2.1 docs and couldn't really tell for sure why you made it. This would limit use to some newer Docker clients, hence the question.